runtime/Makefile

```diff
@@ -81,8 +81,8 @@ BINARY_NAME=sledgert
 
 # Sets a flag equal to the processor architecture
 CFLAGS += -D${ARCH}
-CFLAGS += -DNCORES=${TOTAL_CORES}
-CFLAGS += -DPAGE_SIZE=$(PAGE_SIZE)
+# CFLAGS += -DNCORES=${TOTAL_CORES}
+# CFLAGS += -DPAGE_SIZE=$(PAGE_SIZE)
```

Comment:
Also uncomment this to declare the macro

Reply:
It is being defined in the `types.h` file. Since @gparmer  said it should always be **4096**, I figured reading it from the machine is unnecessary.

Reply:
Cool. Just go ahead and delete these zombie lines from the makefile then! 🙌